src/components/DatepickerRange/index.js

```diff
@@ -0,0 +1,178 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ReactDatePicker, { registerLocale } from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import Icon from '../Icon';
+import {
+  StyledDatepickerInput,
+  StyledDatepicker,
+} from '../DatepickerDay/styles';
+import { StyledRangeDatepicker, StyledUiText } from './styles';
+import { theme } from '../../theme';
+
+const DatepickerRange = ({
+  iconLeft,
+  iconColor,
+  invalid,
+  valid,
+  placeholder,
+  onChange,
+  isClearable,
+  inputProps,
+  dateFormat,
+  locale,
+  withPortal,
+  startPlaceholder,
+  endPlaceholder,
+  datepickerColor,
+  margin,
+  ...props
+}) => {
+  const [startDate, setStartDate] = useState();
+  const [endDate, setEndDate] = useState();
+
+  useEffect(() => {
+    if (locale) registerLocale(locale.key, locale.dateFnsLocale);
+  }, [locale]);
+
+  const DatepickerRangeInput = React.forwardRef((dateInputProps, ref) => {
+    return (
+      <StyledDatepickerInput
+        {...dateInputProps}
+        ref={ref}
+        placeholder={placeholder}
+        icon={Icon.ICONS.IconCalendar}
+        iconLeft={iconLeft}
+        invalid={invalid}
+        valid={valid}
+        borderColor={datepickerColor && theme.color.brand.primary}
+        {...inputProps}
```

Comment:
If there is only 1 input being used twice (with a single `inputProps`) how can the user define id's (or other unique props) for each input?

Reply:
Do you think we will have a situation where we want different props for start and end date picker ? Like a different icon or bordercolor ? (expect Id, which needs to be different)
 Do you want me to have all the props for end and start like the startPlaceholder, so we will have `startIconLeft` `endIconLeft` and so on ?

Reply:
How will the user get `start_date` and `end_date` back from the Flamingo component? Have you tested this with a form to see you are getting the values back?

Reply:
Agathe, maybe the icons, in case we want a different icon for each. But the colors (of the icon, the border...) I don't think so. I don't see why in a design we would want different colors of fields within the same form.